cs-amazon/src/test/java/io/cloudslang/content/amazon/execute/QueryApiExecutorTest.java

```diff
@@ -833,6 +851,12 @@ private LoadBalancerInputs getLoadBalancerInputs() {
                 queryParamsMap.put("SnapshotId", "snap-id");
                 queryParamsMap.put("AvailabilityZone", "us-east-1d");
                 break;
+            case "DescribeVolumes":
+                queryParamsMap.put("Filter.1.Name", "status");
+                queryParamsMap.put("Filter.1.Value.1", "in-use");
```

Comment:
I would suggest to add a filter with 2 values to test the splitting functionality as well.